projects/js-toolkit/packages/portal-adapt-base/package.json

```diff
@@ -1,23 +1,32 @@
 {
+	"author": "Liferay Frontend Infrastructure Team <pt-frontend-infrastructure@liferay.com>",
 	"dependencies": {
 		"@liferay/portal-base": "^1.0.0",
 		"cross-spawn": "^7.0.0",
 		"escape-string-regexp": "^2.0.0",
 		"globby": "^10.0.1"
 	},
 	"description": "Base library for adaptation autopresets",
-	"main": "./src/index.js",
+	"license": "LGPL-3.0",
+	"main": "lib/index.js",
```

Comment:
Just to verify, we will need to re-build this during development anytime we make changes?